Server/EVESharp.Node/Services/Space/ConcreteServiceAttribute.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace EVESharp.Node.Services
+{
+    [AttributeUsage(AttributeTargets.Class)]
+    public sealed class ConcreteServiceAttribute : Attribute
+    {
+        public string ServiceName { get; }
+
+        public ConcreteServiceAttribute(string serviceName)
```

Comment:
What is this attribute for? I see it used in a couple of classes but there's no code that actually expects it...